pages/index.tsx

```diff
@@ -18,178 +20,133 @@ export const getStaticProps: GetStaticProps = async () => {
+              /><span className="text-white"> .</span></h1>        
+          </div>
+          <div className="m-auto mt-6 md:mt-8">
+            Gehälter für Polizist:innen, der Betrieb öffentlicher Gebäude, die Förderungen von kulturellen Einrichtungen, die Beleuchtung des Straßenraums: Das Leben und Zusammenleben in Berlin verursacht viele laufende Kosten. Dazu kommen langfristige Investitionen in Einrichtungen und Infrastruktur, wie öffentliche Schulen und die Verbesserung von Parks. Ausgaben die durch die Stadtverwaltung getragen werden müssen. Doch wofür werden wieviele Gelder ausgegeben?
+            <br></br><br></br>
+            Die Ausgaben sind im sogenannten Berliner Haushalt festgelegt. Sie lassen sich in 9 bundesweit vereinheitlichte Hauptfunktionsbereiche untergliedern. Diese können als oberste Stufe einer Treemap dargestellt werden:
+          </div>   
         </div>
-        <div className="flex-col">
-          <p className="text-sm">
-            Gesamtausgaben des Berliner Haushalts in Mrd. €
-          </p>
-          <GroupedBarChart data={TOTAL_EXPENSES} />
+
+        <div className="md:flex justify-center m-auto mt-6 md:mt-12 ">
+          <div className="flex-col mb-6 md:my-auto inline-block">
+          <FadeInWrapper>
+            <div className="md:pr-12">
+            <ul>
+            <span className="font-bold text-xl text-brand">
+                <InternalLink href={'/visualisierung'}>
+                  {'→ Zur Visualisierung'}
+                </InternalLink>
+              </span>
+              <li><p className="pl-6">Alle Ausgaben und Einnahmen<br></br>in der Tree Map erkunden</p>
+              </li>
+            </ul>
+          </div>
+          </FadeInWrapper>
+          </div>
+          <div className="flex-col inline-block my-auto justify-center md:pr-10">
+            <iframe style={{ width: "100%" }}
+              title="Beispiel-Visualisierung der Haushaltsdaten"
+              width="400rem"
+              height="400"
+              src="https://haushaltsdaten-staging.odis-berlin.de/share"
+            ></iframe>
+          </div>
         </div>
-      </div>
 
-      <FadeInWrapper>
-        <p className="text-4xl flex justify-center mt-16">
-          76,61 Milliarden Euro
-        </p>
-        <p className="text-gray-500 text-sm flex justify-center mb-12">
-          {' '}
-          stehen Berlin im aktuellen Doppelhaushalt als Gesamtausgaben zur
-          Verfügung
-        </p>
-      </FadeInWrapper>
+        <div className="lg:w-3/6 m-auto mt-6 md:mt-12">
+            Im Berliner Haushaltsplan sind die Ausgaben und Einnahmen außerdem spezifischen Bereichen zugeordnet: Der Hauptverwaltung und den zwölf Berliner Bezirken. Außerdem werden die Beträge für den einzelnen Haushaltsjahren getrennt angegeben. Mittels der Visualisierung die über den Link oben erreichbar ist, können die einzelnen Bereiche und Jahre erkundet werden.
+            <br></br><br></br>
```

Comment:
In general, it's semantically better to use two `<p>` tags here instead of the manual line breaks. You can then give the `<p>`'s e.g. a margin-top to create the whitespace.